change/@microsoft-fast-element-ad8e6227-ad64-4f86-9f52-44ea25546893.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "docs: fix a small mistake",
```

This could be anything, can we make the change file a bit more explicit?
```suggestion
  "comment": "docs: add missing closing parenthesis to when directive example",
```

---

Oops… I just selected the title before. Thank you!